package.json

```
@@ -48,7 +48,7 @@
         "marked": "^2.0.3",
         "mz": "^2.4.0",
         "node-notifier": "^9.0.1",
-        "semver": "^7.3.5"
+        "composer-version-constraint-evaluator": "^1.0.0"
```

We should scope the package name to pickware.
```suggestion
        "@pickware/composer-version-constraint-evaluator": "^1.0.0"
```

---

Haven't published yet. I did not know that we have such a scope